scripts/verify.sh

```diff
@@ -37,18 +31,14 @@ golangci-lint run --disable-all \
     --enable=ineffassign \
     --enable=nakedret \
     --enable=interfacer \
-    --enable=misspell \
```

Comment:
Any reason for removing this?

Reply:
duplicated :-)

Reply:
Exactly, it's a duplicate

Reply:
I'd suggest always sort this list, so dup will be very unlikely.

Reply:
Should we wait until all the Golint related PR are merged?

Reply:
That is fine as long as we don't forget to do it.
We can probably track it in the umbrella issue.